src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if (days >= 3 && days < 7) {
+    number = days * 40 - 20;
```

Comment:
You can omit this `else if` block by using `return` statements in the previous `if` blocks. This will make the code cleaner and more straightforward.